helm_drydock/config.py

```diff
@@ -21,21 +21,13 @@
 
 class DrydockConfig(object):
```

Comment:
Agreed. config.py is just a placeholder until I understand how the service should load configuration (file, environment, external service). I anticipate config.py would own the logic to load the configuration and make it available to the rest of Drydock similarly to how it is currently hardcoded. If INI files are the preferred method, I can add an issue to implement parsing config from INI.